blueprints/packages/files/packages/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Generally speaking, we should not be adding `.eslintrc.js` files, instead we should rely on `overrides` configuration in the top level eslintrc.

Reply:
I'm following the pattern of the existing `lib` blueprint. I can follow this PR up with changes to both `lib` and `packages`

Reply:
sounds good